### PR TITLE
Added graphic of available seat miles

### PR DESCRIPTION
**Pull request by rudeboybert**
Showing this graphic really help the students get what an available seat mile is. Also it makes LC3.20 much more prominent. Feel free to push back.

### CODE REVIEW

File: 03-wrangling.Rmd [hunk 1]
```diff
@@ -905,7 +905,13 @@ if(knitr:::is_latex_output()){
...
 
-An airline industry measure of a passenger airline's capacity is the [available seat miles](https://en.wikipedia.org/wiki/Available_seat_miles), which is equal to the number of seats available multiplied by the number of miles or kilometers flown summed over all flights. So, for example, say an airline had 2 flights using a plane with 10 seats that flew 500 miles and 3 flights using a plane with 20 seats that flew 1000 miles, the available seat miles would be 2 $\times$ 10 $\times$ 500 $+$ 3 $\times$ 20 $\times$ 1000 = 70,000 seat miles.
+An airline industry measure of a passenger airline's capacity is the [available seat miles](https://en.wikipedia.org/wiki/Available_seat_miles), which is equal to the number of seats available multiplied by the number of miles or kilometers flown summed over all flights. So, for example, say an airline had 2 flights using a plane with 10 seats that flew 500 miles and 3 flights using a plane with 20 seats that flew 1000 miles, the available seat miles would be 2 $\times$ 10 $\times$ 500 $+$ 3 $\times$ 20 $\times$ 1000 = 70,000 seat miles.
```
Comment (ismayc) [inline 03-wrangling.Rmd]: Is there a reason why the numbers aren't wrapped in the `$` `$` as well? I noticed we aren't consistent in applying this throughout the book. It does take up more space in some situations but LaTeX also won't break apart equations as much if the formula is between `$` `$`.

File: 03-wrangling.Rmd [hunk 2]
```diff
@@ -905,7 +905,13 @@ if(knitr:::is_latex_output()){
...
+An airline industry measure of a passenger airline's capacity is the [available seat miles](https://en.wikipedia.org/wiki/Available_seat_miles), which is equal to the number of seats available multiplied by the number of miles or kilometers flown summed over all flights. So, for example, say an airline had 2 flights using a plane with 10 seats that flew 500 miles and 3 flights using a plane with 20 seats that flew 1000 miles, the available seat miles would be 2 $\times$ 10 $\times$ 500 $+$ 3 $\times$ 20 $\times$ 1000 = 70,000 seat miles. 
+
+For example, let's consider the scenario in Figure \@ref(fig:available-seat-miles). Since the airplanes has 4 seats and it travels 200 miles, the available seat miles are 4 $\times$ 200 = 800.
+
+```{r available-seat-miles, echo=FALSE, fig.cap="Example of available seat miles for one flight.", fig.width=8, purl=FALSE}
+knitr::include_graphics("images/flowcharts/flowchart/flowchart.012.png")
```
Comment (ismayc) [inline 03-wrangling.Rmd]: I like it! That's helpful.

Comment (ismayc) [reply 03-wrangling.Rmd]: I'll merge this in and then sync it back with the `crc-edits` branch where I've been getting the images to appear in the right spots.

File: 03-wrangling.Rmd (hunk 1 above)
Comment (rudeboybert) [reply 03-wrangling.Rmd]: sounds good!

**Comment by rudeboybert** [comment]
Heads up Chester, I changed the branch/PR a bit.

**Pull request merged**